tests/test_cases/test_failure/test_failure.py

```diff
@@ -0,0 +1,9 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+import cocotb
```

Comment:
A docstring explaining the purpose of this file would be good (and mentions that the makefile is special to handle it)